tardis/model.py

```diff
@@ -274,7 +274,13 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
 
         """
         model_path = os.path.join(path, 'model')
-        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
+        properties = [
```

Comment:
Well I saw a `getattr` statement blow, so I added whatever was new was tested here, for it to be stored in reference HDF file. @ftsamis is it happening automatically / easily somewhere else ?